doc/adoc/user_guide.adoc

```diff
@@ -155,8 +155,8 @@ all required repositories to be setup correctly.
 
 Specify Migration Product::
 With the SLES15-Migration package an auto detected product is
-already applied. If that detection failed, the setting can be overwritten with one of the
-following product specifications:
+already applied. If that detection failed, the setting can be overwritten
+with a product specifications in the following format:
```

Comment:
Actually from our e-mail conversation we don't want people to specify the product themselves due to the concerns you raised. So what about deleting the documentation in this regard ?